src/components/Simulator/index.tsx

```diff
@@ -44,14 +47,15 @@ export const Simulator: React.FC<IProps> = ({ children, data, location }) => {
       eventName: `close-from-step-${step + 1}`,
     })
     // Clearing URL Params will reset pathId, and close modal
-    clearUrlParams(location)
+    clearUrlParams()
   }
 
   // Remove URL search param if invalid pathId
   useEffect(() => {
     setStep(0)
-    if (!pathId) clearUrlParams(location)
-  }, [location, pathId])
+    if (!pathId && router.asPath !== "/wallets#sim") clearUrlParams()
```

Comment:
Why do we need to add this new check?

Reply:
Shouldn't it be?
```suggestion
    if (!pathId && router.asPath !== "/wallets") clearUrlParams()
```

Since now, when you load the page, you get auto-scrolled to the #sim section.

Reply:
@wackerow pushed a fix up for this